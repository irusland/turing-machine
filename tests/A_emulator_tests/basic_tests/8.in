2
start stop
start stop
1
start 1 -> stop _ L
1