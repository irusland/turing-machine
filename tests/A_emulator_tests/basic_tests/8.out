STOP after 1 transitions

stop 0
